frontend/screens/BrandAccountReviewNotification.js

```diff
@@ -1,12 +1,31 @@
 import * as React from "react";
-import { StyleSheet, View, Text, TouchableOpacity } from "react-native";
+import { StyleSheet, View, Text, TouchableOpacity, Alert } from "react-native";
 import Depth1Frame5 from "../components/Depth1Frame5";
 import { Padding, FontFamily, FontSize, Color, Border } from "../GlobalStyles";
-import { useNavigation } from "@react-navigation/native";
 
 
-const BrandAccountReviewNotification = () => {
-  const navigation = useNavigation();
+const BrandAccountReviewNotification = ({ route, navigation }) => {
+  const { payload } = route.params;
+
+  const registerBrand = async () => {
+    try {
+      const response = await fetch("http://192.168.160.76:3000/brands/signup", {
```

Comment:
please read the endpoint from environment variable and add it in .env.sample so that we can use hosted endpoint in the deployed app

Reply:
ok